### ISSUE TITLE
No xreg argument on forecast method

### ISSUE THREAD

**Issue by berkorbay** [Nixtla/statsforecast#47]
Hi, thanks for the package! I simply cannot reproduce the error because of the peculiar error: `forecast() got an unexpected keyword argument 'xreg'`. I installed the package from pip for Python3.8. Package version is 0.3.0. I am on a M1 Mac but I am not currently using arm64 version.

I also checked the module with inspect package. It really does not have the "xreg" argument in forecast method. Here are the code and response.

```
import inspect
from statsforecast import StatsForecast
inspect.getfullargspec(StatsForecast.forecast)
```

The response is  `FullArgSpec(args=['self', 'h'], varargs=None, varkw=None, defaults=None, kwonlyargs=[], kwonlydefaults=None, annotations={})`

p.s. I just installed from Github and xreg seems to be there. Probably it will work, too.

**Comment by FedericoGarza** [comment]
Hi @berkorbay! We haven't released yet (`pypi`) the exogenous variables feature, so you are not able to use them. We are planning to release it today! 🎉

**Comment by FedericoGarza** [comment]
Hi @berkorbay! We just released the `0.4.0` version. It includes the `exogenous variables` feature. 🎉 

<img width="983" alt="image" src="https://user-images.githubusercontent.com/10517170/156250055-f1b93362-e2bb-4907-ad98-3e69af1e6fc2.png">

I'm closing this issue but feel free to open another if you have questions or issues running the model.
